actpool/actpool.go

```diff
@@ -429,6 +423,21 @@ func (ap *actPool) removeInvalidActs(acts []action.SealedEnvelope) {
 		delete(ap.allActions, hash)
 		intrinsicGas, _ := act.IntrinsicGas()
 		ap.gasInPool -= intrinsicGas
+
+		//del actions in destination map
+		desAddress, ok := act.Destination()
+		if ok {
+			dst := ap.accountDesActs[desAddress]
+			if dst != nil {
+				for i, v := range ap.accountDesActs[desAddress] {
+					h := v.Hash()
+					if bytes.Equal(h[:], hash[:]) {
+						ap.accountDesActs[desAddress] = append(ap.accountDesActs[desAddress][:i], ap.accountDesActs[desAddress][i+1:]...)
+						break
+					}
+				}
```

Comment:
The deletion will be slow if there's many txs sent to the same destination.

Reply:
I'll find a better way later